hazelcast/src/test/java/com/hazelcast/map/InterceptorTest.java

```diff
@@ -90,6 +88,44 @@ public void testMapInterceptor() throws InterruptedException {
 
     }
 
+    @Test
+    public void testMapInterceptorOnNewMember() throws InterruptedException {
+        TestHazelcastInstanceFactory nodeFactory = createHazelcastInstanceFactory(2);
+        Config config = getConfig();
+        HazelcastInstance instance1 = nodeFactory.newHazelcastInstance(config);
+        IMap map = instance1.getMap("map");
+        for (int i = 0; i < 100; i++) {
+            map.put(i, i);
+        }
+        map.addInterceptor(new NegativeInterceptor());
```

Comment:
this PR is all about refactoring current tests so that enterprise part can use them too, with a different config.
so enhancing current tests is not scope of this PR